### PR TITLE
Update xknx to 0.15.3

### PR DESCRIPTION
**Pull request by marvin-w**
## Proposed change

Documentation update for underlying library change. See core PR for more details.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/42026
- Link to parent pull request in the Brands repository: -
- This PR fixes or closes issue: -

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/knx.markdown [hunk 1]
```diff
@@ -885,6 +895,10 @@ type:
   description: A type from the following table must be defined. The DPT of the group address should match the expected KNX DPT to be parsed correctly.
   required: true
   type: string
+always_callback:
+  description: Defines if each update of the internal state should trigger a callback within Home Assistant no matter if the payload is equal.
```
Comment (farmio) [inline source/_integrations/knx.markdown]:
```suggestion
  description: Defines if telegrams with equal payload as the previously received telegram should trigger a state update within Home Assistant.
```

File: source/_integrations/knx.markdown [hunk 2]
```diff
@@ -282,14 +282,19 @@ device_class:
   required: false
   type: string
 reset_after:
-  description: Reset back to OFF state after specified milliseconds.
+  description: Reset back to OFF state after specified seconds.
   required: false
-  type: integer
+  type: float
+ignore_internal_state:
+  description: Specifies if state updates should ignore the internal state and always trigger the Home Assistant callback.
```
Comment (farmio) [inline source/_integrations/knx.markdown]:
```suggestion
  description: Specifies if telegrams should ignore the internal state and always trigger a Home Assistant state update.
```

**Review by farmio — DISMISSED** [review]

**Review by frenck — DISMISSED** [review]
✅ Approved. Can be merged as soon as the parent PR gets merged.

File: source/_integrations/knx.markdown [hunk 3]
```diff
@@ -283,19 +283,19 @@ device_class:
...
-  description: Reset back to OFF state after specified milliseconds.
+  description: Reset back to OFF state after specified seconds.
   required: false
-  type: integer
+  type: float
 ignore_internal_state:
-  description: If set to true the update callbacks are always executed regardless of the current binary sensor state. If some of your automations are triggered multiple times make sure this setting is False.
-  required: false
+  description: Specifies if telegrams should ignore the internal state and always trigger a Home Assistant state update.
+  required: true
```
Comment (farmio) [inline source/_integrations/knx.markdown]:
```suggestion
  required: false
```
ignore_internal_state is optional

File: source/_integrations/knx.markdown [hunk 4]
```diff
@@ -891,6 +896,10 @@ type:
   description: A type from the following table must be defined. The DPT of the group address should match the expected KNX DPT to be parsed correctly.
   required: true
   type: string
+always_callback:
+  description: Defines if telegrams with equal payload as the previously received telegram should trigger a state update within Home Assistant.
+  type: boolean
```
Comment (farmio) [inline source/_integrations/knx.markdown:907]:
```suggestion
  required: false
  type: boolean
```
required key missing

**Review by farmio — CHANGES_REQUESTED** [review]

**Review by farmio — APPROVED** [review]
👍

File: source/_integrations/knx.markdown [hunk 5]
```diff
@@ -283,19 +283,19 @@ device_class:
   required: false
   type: string
 reset_after:
-  description: Reset back to OFF state after specified milliseconds.
+  description: Reset back to OFF state after specified seconds.
   required: false
-  type: integer
+  type: float
```
Comment (farmio) [inline source/_integrations/knx.markdown:288]:
```suggestion
  type: float
invert:
  description: Invert the telegrams payload before processing. This is applied before `context_timeout` or `reset_after` is evaluated.
  required: false
  type: boolean
  default: False
```

File: source/_integrations/knx.markdown [hunk 6]
```diff
@@ -1078,6 +1088,10 @@ state_address:
   description: Separate KNX group address for retrieving the switch state. *DPT 1*
   required: false
   type: string
+reset_after:
```
Comment (farmio) [inline source/_integrations/knx.markdown:1101]:
```suggestion
invert:
  description: Invert the telegrams payload before processing or sending. This is applied before `reset_after` is evaluated so the reset will send "1" to the bus.
  required: false
  type: boolean
  default: False
reset_after:
```

**Review by farmio — CHANGES_REQUESTED** [review]
add documentation for invert from 0.15.3

File: source/_integrations/knx.markdown [hunk 7]
```diff
@@ -283,19 +283,19 @@ device_class:
...
+  type: float
 ignore_internal_state:
-  description: If set to true the update callbacks are always executed regardless of the current binary sensor state. If some of your automations are triggered multiple times make sure this setting is False.
+  description: Specifies if telegrams should ignore the internal state and always trigger a Home Assistant state update.
   required: false
   type: boolean
-  default: True
+  default: False
 context_timeout:
   description: The time in seconds between multiple identical telegram payloads would count towards the internal counter that is used for automations. Ex. You have automations in place that trigger your lights on button press and another set of lights if you click that button twice. This setting defines the time that a second button press would count toward, so if you set this 3.0 you can take up to 3 seconds in order to trigger the second button press. Maximum value is 10.0.
```
Comment (farmio) [inline source/_integrations/knx.markdown]:
```suggestion
  default: False
context_timeout:
  description: The time in seconds between multiple identical telegram payloads would count towards the internal counter that is used for automations. Ex. You have automations in place that trigger your lights on button press and another set of lights if you click that button twice. This setting defines the time that a second button press would count toward, so if you set this 3.0 you can take up to 3 seconds in order to trigger the second button press. If set `ignore_internal_state` will be set to `True` internally. Maximum value is 10.0.
```

**Review by farmio — CHANGES_REQUESTED** [review]

**Review by farmio — DISMISSED** [review]
👍 now for real 🤣

**Comment by a005** [comment]
This PR should be merged for 0.118 release! It describes breaking change.

**Comment by frenck** [comment]
This PR is missed when creating the 0.118 release. Changed the target branch of this PR to `current` so it ends up in the current documentation of 0.118.

**Review by frenck — APPROVED** [review]
Looks good! Thanks! 👍

**Pull request merged**